<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsdWidget</class>
 <widget class="QWidget" name="OsdWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>70</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowIcon">
   <iconset theme="media-playback-start">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="controls">
     <item>
      <widget class="VlcWidgetSeek" name="seek" native="true"/>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLCDNumber" name="number">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>34</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonPlay">
       <property name="toolTip">
        <string>Play</string>
       </property>
       <property name="statusTip">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonStop">
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="statusTip">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-stop">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonBack">
       <property name="toolTip">
        <string>Back</string>
       </property>
       <property name="statusTip">
        <string>Back</string>
       </property>
       <property name="icon">
        <iconset theme="media-skip-backward">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonNext">
       <property name="toolTip">
        <string>Next</string>
       </property>
       <property name="statusTip">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset theme="media-skip-forward">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonRecordNow">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Record Now</string>
       </property>
       <property name="statusTip">
        <string>Record Now</string>
       </property>
       <property name="icon">
        <iconset theme="media-record">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonSnapshot">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Take snapshot</string>
       </property>
       <property name="statusTip">
        <string>Take snapshot</string>
       </property>
       <property name="icon">
        <iconset theme="image-x-generic">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonTeletext">
       <property name="toolTip">
        <string>Teletext</string>
       </property>
       <property name="statusTip">
        <string>Teletext</string>
       </property>
       <property name="icon">
        <iconset theme="text-x-generic-template">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="teletext">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>34</height>
        </size>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>888</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="VolumeSlider" name="volume">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonMute">
       <property name="toolTip">
        <string>Mute</string>
       </property>
       <property name="statusTip">
        <string>Mute</string>
       </property>
       <property name="icon">
        <iconset theme="audio-volume-muted">
         <normaloff/>
        </iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="3">
    <widget class="QLabel" name="logo"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VlcWidgetSeek</class>
   <extends>QWidget</extends>
   <header location="global">VLCQtWidgets/WidgetSeek.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VolumeSlider</class>
   <extends>QSlider</extends>
   <header>style/VolumeSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
